values.yaml

```diff
@@ -19,6 +19,10 @@ global:
   privateCaCertsAddToHost:
     enabled: false
     hostDirectory: /etc/docker/certs.d
+    addToContainerd: false
+    containerdConfigPath: /etc/containerd/certs.d
```

Comment:
key name should be something else as that sounds like the path that should put to containerd.toml itself. Maybe rename to containerdCertsConfigPath or similar?